frontend/test/metabase/scenarios/question/bookmarks.cy.spec.js

```diff
@@ -0,0 +1,44 @@
+import { restore, sidebar } from "__support__/e2e/cypress";
+import { getSidebarSectionTitle as getSectionTitle } from "__support__/e2e/helpers/e2e-collection-helpers";
+
+describe("scenarios > question > view", () => {
+  beforeEach(() => {
+    restore();
+    cy.signInAsAdmin();
+  });
+
+  describe("bookmarks", () => {
```

Comment:
If this whole spec is dedicated to question bookmarks, why do we need this describe block? Do you plan to add more tests that might have a different context?

If not, the top most `describe` could be `scenarios > question > bookmarks`.